action.yaml

```diff
@@ -1,60 +1,78 @@
 name: "Prepare test"
 description: Performs some preparation to run tests
+
 inputs:
   ADMIN_PAT:
-    description: ADMIN_PAT
+    description: Personal Access Token for admin operations
     required: true
   AWS_ACCESS_KEY_ID:
-    description: AWS_ACCESS_KEY_ID
+    description: AWS Access Key ID
     required: true
   AWS_SECRET_ACCESS_KEY:
-    description: AWS_SECRET_ACCESS_KEY
+    description: AWS Secret Access Key
     required: true
   AWS_REGISTRY:
-    description: AWS_REGISTRY
+    description: AWS ECR Registry
     required: true
+  AWS_REGION:
+    description: AWS region
+    required: false
+    default: "us-east-1"
   VERSION_TAG:
-    description: VERSION_TAG
+    description: Git repository version tag to checkout
     required: false
   REPOSITORY:
-    description: REPOSITORY
+    description: Git repository to checkout
     required: false
   REPO_DEPTH:
-    description: REPO_DEPTH
+    description: Git repository depth to fetch
     required: false
     default: "1"
   NODE_VERSION:
-    description: NODE_VERSION
+    description: Node.js version to setup
     required: false
     default: "20.10"
   DISABLE_NPM_CACHE:
-    description: DISABLE_NPM_CACHE
+    description: Disable npm cache
     required: false
     default: "false"
   SHEETJS_TOKEN:
-    description: SHEETJS_TOKEN
+    description: Token for SheetJS registry
     required: false
     default: ""
   NODE_PACKAGE_MANAGER:
-    description: "Node package manager npm, yarn, or pnpm"
+    description: Node package manager to use (npm, yarn, or pnpm)
     required: false
     default: "npm"
+  USE_BLACKSMITH:
+    description: Use the blacksmith setup actions
+    required: false
+    default: "false"
+
 runs:
   using: composite
   steps:
     - name: Debug
       run: echo "${{ github.repository }}/${{ github.ref_name }}"
       shell: bash
 
-    - name: Checkout Repo
+    - name: Validate Inputs
+      run: |
+        if [[ -z "${{ inputs.VERSION_TAG }}" && -z "${{ inputs.REPOSITORY }}" ]]; then
+          echo "Error: VERSION_TAG or REPOSITORY must be provided."
```

Comment:
Is this true? If neither are supplied, wouldn't it still checkout the latest commit of the current repo

Reply:
you are correct, neither is required.  removing